web/template/create_document.gohtml

```diff
@@ -5,204 +5,155 @@
-                                        <div class="govuk-form-group {{ if .Error.Field.correspondenceBy }}govuk-form-group--error{{ end }}">
-                                        <fieldset class="govuk-fieldset">
-                                            <legend class="govuk-fieldset__legend">Correspondence by</legend>
-                                            {{ template "errors" .Error.Field.correspondenceBy }}
-                                            <div class="govuk-checkboxes" data-module="govuk-checkboxes">
-                                                <div class="govuk-checkboxes__item">
-                                                    <input class="govuk-checkboxes__input" id="f-correspondenceBy-post" name="correspondenceBy" type="checkbox" value="post">
-                                                    <label class="govuk-label govuk-checkboxes__label" for="f-correspondenceBy-post">
-                                                        Post
-                                                    </label>
-                                                </div>
-                                                <div class="govuk-checkboxes__item">
-                                                    <input class="govuk-checkboxes__input" id="f-correspondenceBy-email" name="correspondenceBy" type="checkbox" value="email">
-                                                    <label class="govuk-label govuk-checkboxes__label" for="f-correspondenceBy-email">
-                                                        Email
-                                                    </label>
-                                                </div>
-                                                <div class="govuk-checkboxes__item">
-                                                    <input class="govuk-checkboxes__input" id="f-correspondenceBy-phone" name="correspondenceBy" type="checkbox" value="phone">
-                                                    <label class="govuk-label govuk-checkboxes__label" for="f-correspondenceBy-phone">
-                                                        Phone
-                                                    </label>
-                                                </div>
-                                                <div class="govuk-checkboxes__item">
-                                                    <input class="govuk-checkboxes__input" id="f-correspondenceBy-welsh" name="correspondenceBy" type="checkbox" value="welsh">
-                                                    <label class="govuk-label govuk-checkboxes__label" for="f-correspondenceBy-welsh">
-                                                        Welsh
-                                                    </label>
-                                                </div>
+            {{ if .HasSelectedAddNewRecipient }}
```

Comment:
it might be nice to pull out bits into templates here, you can keep them in the same file and have

```
{{ define "addNewRecipient" }}
  <!-- this content -->
{{ end }}
```

then put `{{ template "addNewRecipient" . }}` like you would for any other